web-react-ts/src/auth/SetPermissions.tsx

```diff
@@ -44,7 +44,7 @@ const SetPermissions = ({
         fellowship: data.memberByEmail?.fellowship.id,
         bacenta: data.memberByEmail?.fellowship?.bacenta?.id,
         council:
-          data.memberByEmail?.fellowship?.bacenta.constituency?.council.id,
+          data.memberByEmail?.fellowship?.bacenta.constituency?.council?.id,
```

Comment:
I don't understand this bit. How would a member not have a council

Reply:
I don't know if it is graphql thing. but I realised while working sometime ago that some of the data is undefined and after some seconds the data loads with values. I am sure thats what is happening